server.js

```diff
@@ -0,0 +1,24 @@
+//Required express, nconf and log4js
```

Comment:
My intention was not to rename index.js to server.js but create a separate server file in that server related code can be moved and invoke that from index file